client/galaxy/scripts/layout/menu.js

```diff
@@ -97,7 +97,12 @@ var Collection = Backbone.Collection.extend({
                     title   : 'Saved Visualizations',
                     url     : 'visualization/list',
                     target  : '_frame'
-            }]
+                },{
+                    title   : 'Galaxy Interactive Environments (GIEs)',
```

Comment:
We'd always called them IEs internally but there seemed to be pressure to name it GIEs